
# Mesh Preprocessing
Theta_pln: 10.0 #controls the planarity of each facet.
Theta_fct: 10.0  #controls overlap of facets  //not used

# random sampling
point_generation_method: random_sample
Distance_bdry: 0.5  # controls distance between contacts and facet boundaries  //not used
Distance_rnn: 2  # controls the radius of contact regions  e.g. 0.5*2 = 0.25 is the real radius


# planners for two-finger gripper
H_max: 0 # stability of the planned grasps
Theta_parl: 0.1 # controls the parallelity of point pair

# set approach direction for two-finger gripper
Approach_boundary: true # set approach direction based on boundary analysis
Approach_rotation: false # set approach direction based on discretized rotation angle and axis
N_da: 5 # rotation angle: N_da angles which equally divided by 360 degrees 
        # rotation axis: the normalized vector between the point pair.  

Stroke_per_jaw: 40 # mm 
gripper_fingers_dist: 20

# display part
display_figure: true
display_cluster: false
display_sample_points: false
display_result_points: true
display_grasp: true
display_cluster_boundary: false
attach_coordination: false

background_color: [1.0, 1.0, 1.0]
mesh_color: [0.5,1,0.9]
point_color: [0.8353,0.7686,0.6314]
grasp_color: [1, 0.5, 0.33]
point_size: 15
gripper_opacity: 0.4

# Gripper parameters

# h: height of the gripper
# w: width of the gripper
# l: length of the gripper

# h_f: height of the gripper finger
# w_f: width of the gripper finger
# l_f: length of the gripper finger

# h_fp: height of the gripper finger pad
# w_fp: width of the gripper finger pad
# l_fp: length of the gripper finger pad

# d_f: 2*d_f is the distance between the 2 fingers along w at beginning

gripper_params: [60, 175, 75, 26, 27, 75, 26, 10, 5, 20]






